index.js

```diff
@@ -1,11 +1,6 @@
 'use strict';
 const {builtinModules} = require('module');
 
-const ignoreList = [
-	'sys'
-];
-
-// eslint-disable-next-line node/no-deprecated-api
-module.exports = (builtinModules || (process.binding ? Object.keys(process.binding('natives')) : []) || [])
-	.filter(x => !/^_|^(internal|v8|node-inspect)\/|\//.test(x) && !ignoreList.includes(x))
```

Comment:
`process.binding` is only necessary for Node.js 4 and below, which is [already not supported by this package](https://github.com/sindresorhus/builtin-modules/blob/0d234f3ad3965c57d8cc5b9ca198c505fe032452/package.json#L14), so I just removed it. Also, the `^internal` regex is also no longer necessary since Node.js 6.